external/grpc/src/core/tsi/ssl_transport_security.cc

```diff
@@ -77,15 +77,25 @@ static const char *pers = "TizenRT";
 #define PEM_BEGIN_CRT           "-----BEGIN CERTIFICATE-----\n"
 #define PEM_END_CRT             "-----END CERTIFICATE-----\n"
 
-#define MBEDTLS_BUF_MAX_SIZE  128
+#define TSI_BUF_SIZE  128
 
 #define DEBUG_LEVEL 0
```

Comment:
The external folder concept was brought to keep the imported code from other repository. This code might be maintained by others and keeps on growing with features and bug fixes. If we modify these imported files and modify it will be difficult to further upgrade to new code version. So 
either 
1, remove code from external, modify and maintain it as part of TizenRT
2. Donot modify the original cloned code, but create interface/wrapper with TizenRT. This will help in upgrading the external source easily

Reply:
I agree that we shouldn't modify sources as much as possible in an external folder. 
But we should modify grpc itself because tizenrt doesn't support full features that linux provides.
We should change security layer because original grpc uses openssl.
So I think that we should keep going on this way for grpc.